lib/liquid/standardfilters.rb

```diff
@@ -151,6 +151,23 @@ def sort_natural(input, property = nil)
       end
     end
 
+    # Sort elements of an array in numeric order
+    # provide optional property with which to sort an array of hashes or drops
+    def sort_numeric(input, property = nil)
+      ary = InputIterator.new(input)
+      if property.nil?
+        ary.sort do |a, b|
+          Utils.to_number(a) <=> Utils.to_number(b)
+        end
+      elsif ary.empty? # The next two cases assume a non-empty array.
+        []
+      elsif ary.first.respond_to?(:[]) && !ary.first[property].nil?
+        ary.sort do |a, b|
+          Utils.to_number(a[property]) <=> Utils.to_number(b[property])
+        end
+      end
```

Comment:
Shall we change `sort` as well?

Reply:
It could result in broken liquid being more obviously broken (i.e. showing `Liquid Error: `...). Perhaps we should add logging for in the `sort` filter so that we will know whether it would break things, then we could turn it into a liquid error if it won't.